lobid/.htaccess

```diff
@@ -0,0 +1,4 @@
+
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^lobid/rpb2 https://raw.githubusercontent.com/lobid/vocabs/master/rpb/rpb2.ttl [R=302,L]
```

Comment:
I think you mean to have ^rpb2$ here since this is already in the lobid dir.  Otherwise it will match /lobid/lobid/rpb2.